### DISCUSSION TITLE
can we use it with google meet,zoom like apps ?

### DISCUSSION THREAD

**Discussion by Greatz08** [hacksider/Deep-Live-Cam#84] [General]
Asking this because i have seen one project where we could do this but the problem was that there was no integration of virtual cam which would render modified face .Does this project also create virtual cam so that we can use that by switching to it when joining google meet,zoom etc ? Avatarify is one project which has this implementation and i tested it long time ago and would expect similar thing in this project too

**Comment by hacksider** [comment]
You can actually use OBS with this. Then just add some filters to make it look better.
![image](https://github.com/user-attachments/assets/64594a83-6ea3-47e1-a5ec-975cc56dbf3e)
Here's the one I'm using to sharpen everything

**Reply by 0769751170** [reply]
Where did you get that Nvidia Super Resolution filter

**Reply by hacksider** [reply]
[you](https://obsproject.com/forum/resources/obs-rtx-superresolution.1890/)

**Comment by seniorengineer0909** [comment]
Hello.
How are you?
Could you explain this problem correctly?
I want to use this app with zoom meeting.
Looking forward to hearing from you as soon as possible.
